src/ts/common/ui/overlay.ts

```diff
@@ -1,22 +1,16 @@
 import "../../../sass/overlay.sass";
 
 const OVERLAY_CLASS_NAME = "better-library-thing-overlay";
-const OVERLAY_ID = "better-library-thing-overlay-id";
 
 const createOverlay = () => {
 	const element = document.createElement("div");
-	element.id = OVERLAY_ID;
 	element.className = OVERLAY_CLASS_NAME;
 	return element;
 };
 
-const removeOverlay = () => {
-	const overlay = document.getElementById(OVERLAY_ID);
-	if (overlay) {
-		document.body.removeChild(overlay);
-	}
```

Comment:
The main fear I have with using an overlay id, is I don't trust us to write perfect enough code that we never have two overlays at the same time. I don't want to rely on an invariant like that

Reply:
very fair, this feel safer and we need to worry about less things so yay